nuxeo-coldstorage/src/main/java/org/nuxeo/coldstorage/io/CanBeSentToColdStorageJsonEnricher.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * (C) Copyright 2023 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Guillaume Renard
+ */
+package org.nuxeo.coldstorage.io;
+
+import java.io.IOException;
+
+import org.nuxeo.ecm.core.api.DocumentModel;
+import org.nuxeo.ecm.core.io.marshallers.json.enrichers.AbstractJsonEnricher;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+
+/**
+ * 
+ * @since TODO
```

Comment:
2023.2? Or the server version, aka 2023.5 probably.

Reply:
oops wan't mean to be pushed sorry